ui/desktop/app/templates/application.hbs

```diff
@@ -86,53 +86,58 @@
 
 <PendingConfirmations as |confirmation accept deny|>
   {{#if confirmation.options.isConnectError}}
-    <Rose::Dialog
-      @heading={{t 'states.error'}}
-      @dismissButtonText={{t 'actions.ok'}}
-      @icon='flight-icons/svg/x-square-16'
-      @style='error'
-      @modal={{true}}
-      @dismiss={{deny}}
-      as |dialog|
-    >
-      <dialog.body>
-        <p>
+    <Hds::Modal @color='critical' as |M|>
+      <M.Header @icon='alert-diamond' @onDismiss={{deny}}>
+        {{t 'states.error'}}
+      </M.Header>
+      <M.Body>
+        <Hds::Text::Body @tag='p'>
           {{confirmation.text}}
-        </p>
-        <p>
+        </Hds::Text::Body>
+        <br />
```

Comment:
Nice work!

small nit: can we please use [Hds::Separator](https://helios.hashicorp.design/components/separator) in place of `<br />`?

Reply:
Oh, actually disregard! I was thinking about the `<hr />` instead.